lib/dry/system/plugins/dependency_graph.rb

```diff
@@ -34,10 +34,10 @@ def self.dependencies
         # @api private
         def register(key, contents = nil, options = {}, &block)
           super
-
-          unless config.ignored_dependencies.include?(key.to_sym)
```

Comment:
Alright, so `ignored_dependencies` should also be an array of strings?

Reply:
Yes definitely! dry-system allows strings as identifiers in auto-injection modules BUT they are all coerced to strings internally, and that's how **dry-container** stores items, so this has to be consistent

Reply:
Ok, that makes things clear to me. I'll fix this today.